jkube-kit/jkube-kit-wildfly-jar/src/test/java/org/eclipse/jkube/wildfly/jar/enricher/WildflyJARHealthCheckEnricherTest.java

```diff
@@ -53,9 +51,10 @@ class WildflyJARHealthCheckEnricherTest {
 
     @BeforeEach
     public void setUp() {
-        context = mock(JKubeEnricherContext.class, RETURNS_DEEP_STUBS);
-        project = mock(JavaProject.class);
-        when(context.getProject()).thenReturn(project);
+       project = JavaProject.builder().build();
+       context = JKubeEnricherContext.builder()
+               .project(context.getProject())
```

Comment:
context is not initialized at this point.
```suggestion
               .project(project)
```